__tests__/loader.spec.js

```diff
@@ -96,4 +97,19 @@ describe('purgecss loader', () => {
         expect(output).toContain('isUsed');
       });
   });
+
+  it('should not strip global classes', () => {
```

Comment:
this can be a async function, then you can drop the use of `expect.assertions(2);` and use await

Reply:
fixed in https://github.com/americanexpress/purgecss-loader/pull/9/commits/3fa1871140eb4d07fb7474db15a3966b1390264a 👍